service/src/PlatformIOImp.hpp

```diff
@@ -117,6 +125,8 @@ namespace geopm
             std::vector<std::shared_ptr<IOGroup> > find_signal_iogroup(const std::string &signal_name) const;
             /// @brief Look up the IOGroup that provides the given control.
             std::vector<std::shared_ptr<IOGroup> > find_control_iogroup(const std::string &control_name) const;
+            bool is_control_split_uniform(const std::string &control_name) const;
+            void adjust_combined(int control_idx, double setting);
```

Comment:
If we are going to reorder the vtable, why not put adjust_combined next to sample_combined.